bzzeth/wire.go

```diff
@@ -33,3 +34,9 @@ var Spec = &protocols.Spec{
 type Handshake struct {
 	ServeHeaders bool // indicates if this node is expected to serve requests for headers
 }
+
+// Dummy message to send from another Swarm node
```

Comment:
we must  not introduce another message, just use the Handshake

We need to test if the protocol returns when 
- a message is sent (currently in this PR it is only the Handshake message)
- the peer disconnects 

I think we should test both.